libs/ledger-live-common/src/families/ethereum/synchronisation.ts

```diff
@@ -77,6 +77,27 @@ export const getAccountShape: GetAccountShape = async (infoInput, { blacklistedT
 
   // transform transactions into operations
   let newOps = flatMap(txs, txToOps({ address, id: accountId, currency }));
+
+  // keep the date of the pending operations that appears in newOps
+  const pendingOperationsDate = {};
+
+  (initialAccount?.pendingOperations || []).concat(initialAccount?.operations || []).forEach(op => {
```

Comment:
Wouldn't just a global if initialAccount do the same as those `|| []` ?

Reply:
yes, good point. I've done a little refactoring now